Ktisis/Helpers/PoseHelpers.cs

```diff
@@ -53,6 +54,7 @@ internal class PoseHelpers {
 
 		public unsafe static void ImportPose(Actor* actor, List<string> path, PoseMode modes) {
 			var content = File.ReadAllText(path[0]);
+			if (path[0].Contains(".cmp")) content = LegacyPoseHelpers.ConvertLegacyPose(content);
```

Comment:
I recommend using the [Path.GetExtension](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getextension) method for this